src/constants.rs

```diff
@@ -1,10 +1,24 @@
 pub const KUBEWARDEN_CUSTOM_SECTION_METADATA: &str = "io.kubewarden.metadata";
+
 pub const KUBEWARDEN_ANNOTATION_POLICY_TITLE: &str = "io.kubewarden.policy.title";
 pub const KUBEWARDEN_ANNOTATION_POLICY_DESCRIPTION: &str = "io.kubewarden.policy.description";
 pub const KUBEWARDEN_ANNOTATION_POLICY_AUTHOR: &str = "io.kubewarden.policy.author";
 pub const KUBEWARDEN_ANNOTATION_POLICY_URL: &str = "io.kubewarden.policy.url";
+pub const KUBEWARDEN_ANNOTATION_POLICY_OCIURL: &str = "io.kubewarden.policy.ociUrl";
 pub const KUBEWARDEN_ANNOTATION_POLICY_SOURCE: &str = "io.kubewarden.policy.source";
 pub const KUBEWARDEN_ANNOTATION_POLICY_LICENSE: &str = "io.kubewarden.policy.license";
 pub const KUBEWARDEN_ANNOTATION_POLICY_USAGE: &str = "io.kubewarden.policy.usage";
 
+pub const KUBEWARDEN_ANNOTATION_ARTIFACTHUB_RESOURCES: &str = "io.artifacthub.resources";
+pub const KUBEWARDEN_ANNOTATION_ARTIFACTHUB_DISPLAYNAME: &str = "io.artifacthub.displayName";
+pub const KUBEWARDEN_ANNOTATION_ARTIFACTHUB_KEYWORDS: &str = "io.artifacthub.keywords";
+pub const KUBEWARDEN_ANNOTATION_RANCHER_HIDDENUI: &str = "io.rancher.hidden-ui";
```

Comment:
I wanted to rename to:

```suggestion
pub const KUBEWARDEN_ANNOTATION_RANCHER_HIDDENUI: &str = "io.kubewarden.hidden-ui";
```

Together with
```
pub const ARTIFACTHUB_ANNOTATION_RANCHER_HIDDENUI: &str = "kubewarden/hidden-ui";
```

In preparation for a standalone UI. This means changes in UI, policies, RFC, but I wonder if it's worth it.

Reply:
Done in https://github.com/kubewarden/policy-evaluator/pull/246/commits/b5a525f5a705fc045d3966e948b29e1f745e2fe5,
also https://github.com/kubewarden/rfc/pull/19.

Reply:
@jordojordo would it be possible to honour https://github.com/kubewarden/rfc/pull/19 in the UI?

Reply:
@viccuad This has been accomplished with [this pr](https://github.com/kubewarden/ui/pull/264)

Reply:
Thanks @jordojordo !